build_tools/docker/gradle-android/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+# An image for cross-compiling IREE's TFLite Java Bindings with Gradle and
+# CMake.
+
+FROM gcr.io/iree-oss/cmake@sha256:9d9953acf5ca0cf1ff3e8de32f10f24dfab1c4e8ec5d1fc047f556024ee4bed6
+
+# Tools for downloading/installing packages.
+RUN apt-get update \
+  && apt-get install -y \
+  unzip \
+  wget
+
+# Download and install openjdk-11
+ARG JDK_VERSION=11
+WORKDIR /install-jdk
+RUN apt-get update && apt-get install -y openjdk-11-jdk
+
+# Download and install Gradle
+ARG GRADLE_VERSION=7.1.1
+ARG GRADLE_DIST=bin
+WORKDIR /install-gradle
+RUN cd /opt && \
+    wget -q https://services.gradle.org/distributions/gradle-${GRADLE_VERSION}-${GRADLE_DIST}.zip && \
+    unzip gradle*.zip && \
+    rm gradle*.zip && \
+    ln -s /opt/gradle-${GRADLE_VERSION}/bin/gradle /usr/bin/gradle
+
+# Download and install Android SDK
+# Note: Uses the latest SDK version from https://developer.android.com/studio,
+# however Gradle will automatically download any additional SDK/tooling versions
+# as necessary.
+ARG ANDROID_SDK_VERSION=7583922
+ENV ANDROID_SDK_ROOT /opt/android-sdk
+RUN mkdir -p ${ANDROID_SDK_ROOT}/cmdline-tools && \
+    wget -q https://dl.google.com/android/repository/commandlinetools-linux-${ANDROID_SDK_VERSION}_latest.zip && \
+    unzip *tools*linux*.zip -d ${ANDROID_SDK_ROOT}/cmdline-tools && \
+    mv ${ANDROID_SDK_ROOT}/cmdline-tools/cmdline-tools ${ANDROID_SDK_ROOT}/cmdline-tools/tools && \
+    rm *tools*linux*.zip
+
+# Accept the license agreements of the SDK components
+RUN yes | ${ANDROID_SDK_ROOT}/cmdline-tools/tools/bin/sdkmanager --licenses
```

Comment:
The recommend way from stackoverflow :D https://stackoverflow.com/questions/38096225/automatically-accept-all-sdk-licences

The sdkmanager does not provide a flag or option to auto accept the licenses automatically. If you try to run any command without the license, it will blurt license spam and not run the command until you submit "yes" to multiple license prompts (and you can't do them all at once).